scene/3d/vehicle_body_3d.cpp

```diff
@@ -399,7 +399,6 @@ real_t VehicleBody3D::_ray_cast(int p_idx, PhysicsDirectBodyState3D *s) {
 	Vector3 source = wheel.m_raycastInfo.m_hardPointWS;
 	wheel.m_raycastInfo.m_contactPointWS = source + rayvector;
 	const Vector3 &target = wheel.m_raycastInfo.m_contactPointWS;
-	source -= wheel.m_wheelRadius * wheel.m_raycastInfo.m_wheelDirectionWS;
```

Comment:
Why is this line removed? (genuine question, it looks like it's the main part of the fix and I don't know this code enough to evaluate the change).

It would generally help to describe a bit more what the fix is in the PR description and the commit message to help for review and future contributions.

Reply:
You are right Sir :+1: , that was the main part of the fix. To know more information about this fix and to know more about VehicleBody you can visit my 
GSoC proposal :
https://docs.google.com/document/d/1P6wNMr-bsVEsX_Eh4-SKHb_rfNGzdujDs-0ldeD_45Q/edit#heading=h.g5s3esky2w5f. 
Can also do a review!!
Tnx :smiling_face_with_three_hearts: !!